css/sections/trial.css

```
@@ -1,427 +1,427 @@
+  justify-content: center;
+  word-break: break-word;
+  white-space: nowrap;
+  border-radius: var(--space-sm);
+  border: 1px solid #0000;
+  height: auto;
+  text-decoration: none;
 }
 
 .windows-download svg {
-	fill: currentColor;
+  fill: currentColor;
```

Stylelint (17.2.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)